t/op/sprintf2.t

```diff
@@ -684,6 +684,13 @@ for my $t (@hexfloat) {
             }
         }
     }
+    if (!$ok && $^O eq "netbsd" && $t->[1] eq "exp(1)") {
+      SKIP:
+        {
+            skip "NetBSD's expl() is just exp() in disguise", 1;
```

Comment:
Should that be `exp1()` rather than `expl()`, *i.e.,* a numeral one rather than the letter 'l'?

Reply:
No, the long double version of exp() is expl() (l for long).